js/jsmodules.go

```diff
@@ -46,13 +46,16 @@ func getInternalJSModules() map[string]interface{} {
 			"k6/experimental/timers is now part of the k6 core, please change your imports to use k6/timers instead."+
 				" The k6/experimental/timers will be removed in k6 v0.52.0"),
 		"k6/experimental/tracing": tracing.New(),
-		"k6/experimental/browser": browser.NewSync(),
-		"k6/experimental/fs":      fs.New(),
-		"k6/net/grpc":             grpc.New(),
-		"k6/html":                 html.New(),
-		"k6/http":                 http.New(),
-		"k6/metrics":              metrics.New(),
-		"k6/ws":                   ws.New(),
+		"k6/experimental/browser": newWarnExperimentalModule(browser.NewSync(),
+			"k6/experimental/browser is now part of the k6 core, please change your imports to use k6/browser instead."+
+				" The k6/experimental/browser will be removed in k6 v0.54.0"),
```

Comment:
Is this official? I don't think we've properly discussed when it will be removed. Could this be changed to something like "...The k6/experimental/browser is planned to be removed in k6 v0.54.0, but may be delayed"?

Reply:
Isn't the migration from `k6/experimental/browser` to `k6/browser` breaking? 🤔 

I think we usually leave a message like that when we graduate a module, but in such cases there are no changes other than the import path, but in this case I don't think that just changing the import is enough, right?

Reply:
@andrewslotin I'd like to hear your thoughts about this as well. Thanks!

Reply:
@joanlopez Yes, it's breaking change. We want to give users some room to adjust their scripts. I'm open to your ideas.

Reply:
I don't have any particular opinion because I don't know if that happened before and how that was handled in such case, if so. But I guess that a short message with a reference for context/instructions on how to migrate, at least, would be nice to have.

Reply:
Yeah, I would second @joanlopez. You have to mention the fact they have to await or handle the returned promise. Potentially, you can directly link the documentation where you are explaining and guiding the to-do steps.

Reply:
I agree with @joanlopez on providing a link to a migration doc here. As for the version, I'd suggest that we rather use a date here, since this provides a clearer timeframe when users need to take action. For example this is what we used as a warning while deprecating the legacy k6 Docker image:

> **THIS IMAGE IS DEPRECATED and its support will be discontinued after Dec 31, 2023. This image has been replaced by [`grafana/k6`](https://hub.docker.com/r/grafana/k6).<br>
Please update your scripts to use [`grafana/k6`](https://hub.docker.com/r/grafana/k6) to continue using the latest version of k6.**

Reply:
To give users enough time to migrate, we want to go with the date September 23rd (Cycle 23 k6 release date).

Reply:
WDYT about this warning message?

> k6/experimental/browser is now part of the k6 core, please change your imports to use k6/browser instead. The k6/experimental/browser will be removed on or after 23rd of September, please ensure your scripts have been migrated to k6/browser by that date. Please also see the migration guide at https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/ for more information on how to migrate your scripts.

[8b59034](https://github.com/grafana/k6/pull/3793/commits/8b59034a30f2ea975880b738542b7faf17b23a83)

Reply:
Much more clear @inancgumus, looks good to me! 👍🏻